src/LinuxTracing/GpuTracepointVisitorTest.cpp

```diff
@@ -48,12 +48,12 @@ class MockTracerListener : public TracerListener {
 
 class GpuTracepointVisitorTest : public ::testing::Test {
  protected:
-  void SetUp() override { visitor_.SetListener(&mock_listener_); }
+  void SetUp() override {}
 
   void TearDown() override {}
```

Comment:
Nit: I think you could just remove those. They aren't pure virtual and using constructors/destructors over SetUp/TearDown when possible is encouraged as well (http://google.github.io/googletest/faq.html#CtorVsSetUp)

Reply:
Yep, done.